apps/api/src/app/auth/auth.module.ts

```diff
@@ -1,7 +1,7 @@
 import { MiddlewareConsumer, Module, NestModule, Provider, RequestMethod } from '@nestjs/common';
 import { JwtModule } from '@nestjs/jwt';
 import { PassportModule } from '@nestjs/passport';
-import * as passport from 'passport';
+const passport = require('passport');
```

Comment:
Broken with SWC.

Reply:
Yes, it's because Passport is CJS only.

We need to make a decision about moving to pure ESM in the repo and a lot of these CJS-only packages will either:
1. need upgrades
2. replacements entirely for small things
3. Or, we wait for EMS compatibility, or port our own patch